CONTRIBUTING_GUIDE.adoc

```diff
@@ -13,7 +13,7 @@ toc::[]
 
 === Notes on Code Contributions
 
-Both projects, devon and OASP, are intended to be easy to contribute to. One service allowing such simplicity is GitHub was therefore selected as preferred collaboration platform.
+Both projects, devon and DEVON, are intended to be easy to contribute to. One service allowing such simplicity is GitHub was therefore selected as preferred collaboration platform.
```

Comment:
devon and DEVON is a nonsense. Only use devonfw. We sent the new branding strategy some days ago. Please take a look to it and fix this.